js/pihole/index.js

```diff
@@ -110,11 +146,42 @@ function updateSummaryData(runOnce) {
 
 function updateQueriesOverTime() {
     $.getJSON("api.php?overTimeData", function(data) {
+        // Add data for each hour that is available
         for (hour in data.ads_over_time) {
-            timeLineChart.addData([data.domains_over_time[hour], data.ads_over_time[hour]], hour + ":00");
+            // Add x-axis label
+            timeLineChart.data.labels.push(hour + ":00");
+            timeLineChart.data.datasets.forEach(function(dataset, index) {
+                // Add data to line with index 0 or 1
+                if(index == 0) dataset.data.push(data.domains_over_time[hour]);
+                if(index == 1) dataset.data.push(data.ads_over_time[hour]);
+            });
```

Comment:
This `forEach` can be rewritten as this (I think, unless I'm missing something):
```
timeLineChart.data.datasets[0].data.push(data.domains_over_time[hour]);
timeLineChart.data.datasets[1].data.push(data.ads_over_time[hour]);
```